architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -277,6 +277,15 @@ external registry. It uses standard docker pull specification for its name,
 eg `openshift/ruby-20-centos7:2.0`. When no tag is specified it is assumed
 the `latest` will be used.
 
+NOTE: When looking at an example image stream definition, say like the
+link:https://github.com/openshift/origin/blob/master/examples/image-streams/image-streams-centos7.json[example centos image streams]
+at the Origin repository on GitHub, you will notice that it will contain definitions of
+ImageStreamTag's and DockerImage's, but not ImageStreamImage's.  This stems
```

Comment:
no apostrophes in plurals.

and it doesn't contain definitions of DockerImages, it contains references to DockerImages.